### PR TITLE
Wind Chill Index

### LINKED ISSUE #666: Add wind chill indicator
### Description

Add wind chill indicator

- https://www.canada.ca/en/environment-climate-change/services/weather-health/wind-chill-cold-weather/wind-chill-index.html
- https://weather.gc.ca/windchill/wind_chill_e.html
- https://en.wikipedia.org/wiki/Wind_chill

### PR DESCRIPTION
**Pull request by aulemahal**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #666 (:smiling_imp: )
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated (with summary of main changes)
- [ ] `bumpversion (major / minor / patch)` has been called on this branch
- [ ] Tags have been pushed (`git push --tags`)

* **What kind of change does this PR introduce?** 
Adds the Wind Chill Index (indices+indicator).

* **Does this PR introduce a breaking change?** 
No.

* **Other information**:
The canadian (ECCC) computation is slightly different from the american (NOAA-NWS) one. The main difference is for weak winds (sfcWind < 5 km/h) : the ECCC defines a second equation while the NWS simply says that (sfcWind < 3 mph) is invalid. Moreover, the maximum valid temperature is 0 °C and jumps to 50 °F when you pass the border.

Instead of having named "methods", I splitted the whole behaviour into three keyword arguments. Might be to complex?
Also, the `min_wind_thresh` might be superfluous as 3 mph is near 5 km/h (but not equal). 

I found no academic references for this, which is quite strange. Googling only led me to FAQs and glossaries, documents orientend for the general public and un-permalinkable web pages...

### CODE REVIEW

File: xclim/indices/_conversion.py
Comment (huard) [inline xclim/indices/_conversion.py:583]: I feel this is a tad too complicated. It's not yet clear to me how to simplify this, but I think we should come up with something less flexible but simpler, ie method={"US", "CAN"}

File: xclim/testing/tests/test_atmos.py
Comment (huard) [inline xclim/testing/tests/test_atmos.py:163]: Expected values from the online calculator ? Just add a comment about where you got them.

**Review by huard — CHANGES_REQUESTED** [review]
Looks good, but let's wait for Emilia's feedback before merging.

File: xclim/indices/_conversion.py
Comment (aulemahal) [reply xclim/indices/_conversion.py:583]: Ok. I was trying not to have this kind of switch, but it's true that 3 kwargs is a bit complicated.

I'd keep `mask_invalid`, but can replace the other 3 by `method`.

File: xclim/testing/tests/test_atmos.py
Comment (aulemahal) [reply xclim/testing/tests/test_atmos.py:163]: The online calculator rounds the output, so it's hard to use it directly here. I check by hand that xclim was returning values corresponding to the calculator, but the numbers themselves are self-computed, so it's more of a continuity test (to check if a change breaks it)...

Comment (aulemahal) [reply xclim/testing/tests/test_atmos.py:163]: Ok found a way to alter the online calculator and remove the rounding. `test_indices` uses those expected values. The atmos test is left as-is.

**Comment by aulemahal** [comment]
@huard  Changes were made following the advice of Emilia.

File: xclim/indices/_conversion.py
Comment (huard) [inline xclim/indices/_conversion.py:558]:
```suggestion
    two equations exist, the conventional one and one for slow winds (usually < 5 km/h), see Notes.
```

**Review by huard — APPROVED** [review]

**Pull request merged**
